homeassistant/components/garmin_connect/const.py

```diff
@@ -136,8 +140,8 @@
         None,
         False,
     ],
-    "averageStressLevel": ["Avg Stress Level", "", "mdi:flash-alert", None, True],
-    "maxStressLevel": ["Max Stress Level", "", "mdi:flash-alert", None, True],
+    "averageStressLevel": ["Avg Stress Level", "lvl", "mdi:flash-alert", None, True],
```

Comment:
If the changes to the unit of measurements aren't needed for the library change we should do them separately top.